<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travelling Salesman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_addVertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteVertex">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить вершину</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="matrix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_updateButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TSP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TSP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DFS">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DFS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BFS">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BFS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
